tools/ports/sdl2_mixer.py

```diff
@@ -30,11 +30,28 @@ def create():
     shutil.copytree(source_path, dest_path)
 
     final = os.path.join(dest_path, libname)
-    ports.build_port(dest_path, final, [], ['-DOGG_MUSIC', '-O2', '-s', 'USE_VORBIS=1', '-s', 'USE_SDL=2'],
-                     ['dynamic_flac', 'dynamic_fluidsynth', 'dynamic_mod', 'dynamic_modplug', 'dynamic_mp3',
-                      'fluidsynth', 'load_mp3', 'music_cmd', 'music_flac', 'music_mad', 'music_mod',
-                      'music_modplug', 'playmus.c', 'playwave.c'],
-                     ['external', 'native_midi', 'timidity'])
+    ports.build_port(
+      dest_path,
+      final,
+      includes=[],
+      flags=[
+        '-s', 'USE_SDL=2',
+        '-s', 'USE_MPG123=1',
```

Comment:
We probably don't want to always build mp3 support here.  Maybe we can make two version of the mixer library?

Reply:
I could follow the pattern on SDL2_IMAGE_FORMATS:
```
// Formats to support in SDL2_mixer. Valid values: ogg, mp3
var SDL2_MIXER_FORMATS = [];
```
Should I put *ogg* as a default, to maintain compatibility?
What do you think?

Reply:
I think thats a good idea yes.  Most people don't want more than once audio format for a given project I imagine.

Reply:
done! :sparkles: